<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>
SPDX-FileCopyrightText: none
SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-KDE-Accepted-GPL
 </author>
 <class>ProjectSettings_UI</class>
 <widget class="QDialog" name="ProjectSettings_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="toolTip">
          <string> Used to store proxy clips, thumbnails, previews</string>
         </property>
         <property name="whatsThis">
          <string> The project folder is the place where assets like proxy clips, thumbnails or previews get stored.</string>
         </property>
         <property name="text">
          <string>Project folder:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="default_folder">
         <property name="text">
          <string>Default folder</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="same_folder">
         <property name="text">
          <string>Parent folder of the project file</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="custom_folder">
           <property name="text">
            <string>Custom:</string>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KUrlRequester" name="project_folder">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="KSeparator" name="kseparator_3"/>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QGroupBox" name="profile_box">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="KSeparator" name="kseparator"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_video_tracks">
         <property name="text">
          <string>Video tracks:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QSpinBox" name="video_tracks"/>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_audio_tracks">
         <property name="text">
          <string>Audio tracks:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QSpinBox" name="audio_tracks"/>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_audio_channels">
         <property name="text">
          <string>Audio channels:</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QComboBox" name="audio_channels">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>2 Channels (Stereo)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4 Channels</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6 Channels</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="label_thumbnails">
         <property name="text">
          <string>Thumbnails:</string>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QCheckBox" name="video_thumbs">
         <property name="text">
          <string>Video</string>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QCheckBox" name="audio_thumbs">
         <property name="text">
          <string>Audio</string>
         </property>
        </widget>
       </item>
       <item row="21" column="0">
        <widget class="QLabel" name="label_25">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Timeline Preview:</string>
         </property>
        </widget>
       </item>
       <item row="21" column="1">
        <layout class="QHBoxLayout" name="preview_profile_box"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Proxy</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="proxy_box">
         <property name="title">
          <string>Proxy clips</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <widget class="QCheckBox" name="generate_proxy">
            <property name="text">
             <string>Generate for videos larger than</string>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QSpinBox" name="proxy_imagesize">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="suffix">
             <string>pixels</string>
            </property>
            <property name="minimum">
             <number>200</number>
            </property>
            <property name="maximum">
             <number>100000</number>
            </property>
            <property name="value">
             <number>800</number>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QCheckBox" name="generate_imageproxy">
            <property name="text">
             <string>Generate for images larger than</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QSpinBox" name="proxy_imageminsize">
            <property name="suffix">
             <string>pixels</string>
            </property>
            <property name="minimum">
             <number>500</number>
            </property>
            <property name="maximum">
             <number>100000</number>
            </property>
            <property name="value">
             <number>2000</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_24">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Encoding profile:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QPlainTextEdit" name="proxyparams">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="11" column="2">
           <layout class="QFormLayout" name="formLayout_3">
            <property name="rowWrapPolicy">
             <enum>QFormLayout::DontWrapRows</enum>
            </property>
            <item row="5" column="1">
             <widget class="QLineEdit" name="le_suffix_proxy">
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="l_suffix_proxy">
              <property name="text">
               <string>Suffix of proxy:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="le_prefix_proxy">
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="l_prefix_proxy">
              <property name="text">
               <string>Prefix of proxy:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="le_relPathProxyToOrig">
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="l_relPathProxyToOrig">
              <property name="text">
               <string>Relative path from proxy to clip:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="le_suffix_clip">
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="l_suffix_clip">
              <property name="text">
               <string>Suffix of clip:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="le_prefix_clip">
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="l_prefix_clip">
              <property name="text">
               <string>Prefix of clip:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="l_relPathOrigToProxy">
              <property name="text">
               <string>Relative path from clip to proxy:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="le_relPathOrigToProxy">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="9" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>External proxy clips:</string>
            </property>
           </widget>
          </item>
          <item row="10" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QComboBox" name="external_proxy_profile"/>
            </item>
            <item>
             <widget class="QToolButton" name="manage_external">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset theme="configure"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Proxy video resize (width)</string>
            </property>
           </widget>
          </item>
          <item row="9" column="2">
           <widget class="QCheckBox" name="external_proxy">
            <property name="text">
             <string>Enable</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="proxy_minsize">
            <property name="suffix">
             <string>pixels</string>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="3">
           <widget class="KSeparator" name="kseparator_2"/>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="proxy_resize">
            <property name="suffix">
             <string>pixels</string>
            </property>
            <property name="minimum">
             <number>200</number>
            </property>
            <property name="maximum">
             <number>100000</number>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLabel" name="image_label">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Proxy image size</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0" colspan="3">
           <widget class="KSeparator" name="kseparator_4"/>
          </item>
          <item row="3" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="KComboBox" name="proxy_profile">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="checkProxy">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="proxy_showprofileinfo">
              <property name="text">
               <string>...</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="proxy_manageprofile">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="KSeparator" name="kseparator_5"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Guides</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Metadata</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QTreeWidget" name="metadata_list">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>2</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QToolButton" name="add_metadata">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="delete_metadata">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Project Files</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="4" column="0">
        <widget class="QLabel" name="label_fonts">
         <property name="text">
          <string>Fonts</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="5">
        <widget class="QListWidget" name="fonts_list">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="5">
        <widget class="QTreeWidget" name="files_list">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="3" colspan="2">
        <widget class="KTreeWidgetSearchLine" name="list_search"/>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="unused_size">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="used_size">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Project files:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="button_export">
         <property name="text">
          <string>Plain Text Export...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="files_count">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="used_count">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Unused clips:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="delete_unused">
         <property name="text">
          <string>Delete files</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Clips used in project:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="unused_count">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="5">
        <widget class="KMessageWidget" name="file_message"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KTreeWidgetSearchLine</class>
   <extends>QLineEdit</extends>
   <header>ktreewidgetsearchline.h</header>
  </customwidget>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
  </customwidget>
  <customwidget>
   <class>KSeparator</class>
   <extends>QFrame</extends>
   <header>kseparator.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>project_folder</tabstop>
  <tabstop>generate_proxy</tabstop>
  <tabstop>metadata_list</tabstop>
  <tabstop>add_metadata</tabstop>
  <tabstop>delete_metadata</tabstop>
  <tabstop>delete_unused</tabstop>
  <tabstop>list_search</tabstop>
  <tabstop>files_list</tabstop>
  <tabstop>fonts_list</tabstop>
  <tabstop>button_export</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectSettings_UI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectSettings_UI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_folder</sender>
   <signal>toggled(bool)</signal>
   <receiver>project_folder</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
